tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -178,8 +152,8 @@ private function lockFileInSpace(
 	 *
 	 * @return void
 	 */
-	public function lockFileUsingWebDavAPI(string $user, string $file, TableNode $properties) {
-		$response = $this->lockFile($user, $file, $properties, false, false);
+	public function tryToLockFileUsingWebDavAPI(string $user, string $file, TableNode $properties) {
```

Comment:
we want to resemble function to the step (visually separates them from helper functions)
```suggestion
	public function userTriesToLockFileSettingPropertiesUsingWebDavAPI(string $user, string $file, TableNode $properties) {
```